portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,72 +44,9 @@ public void init(){
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // System.out.println("Calls doGet"); DEBUG Tool
```

Comment:
Who LGTM'd this commented out code?

Reply:
Dunno, it's been there for a while though.